playwright/base.ts

```diff
@@ -15,6 +20,10 @@ export const test = base.extend<{ failOnJSError: boolean; displayAsciiTable: boo
         const errors: Array<Error> = [];
         const logs: Log[] = [];
 
+        /* throw Error() -  raises an exception in the current code block and causes it to exit, or to flow to the next catch statement if raised in a try block. */
```

Comment:
Are these comments needed?

Reply:
@shoumaw I'm happy to remove these comments, but I was thinking some people might be confused about why we are not failing the tests for console errors but only for throw Error(). So, these comments make it clear.

Reply:
I do not see though any instances of `throw Error()` or `console.error` in the code.
Are those for future usage?

Reply:
@shoumaw I think these comments are creating more confusion 😅 I have removed them.